app/assets/javascripts/backbone/views/timeline_select_steps_view.coffee

```diff
@@ -1,6 +1,6 @@
 class MS.Views.TimelineSelectStepsView extends Backbone.View
 
-  el: 'body'
+  el: '#timeline-header'
```

Comment:
How does this even work? The checkboxes are not in the header, right? So then it should not be possible to affect them from this view unless you're using unscoped `$`, which is a big no-no...